synapse/lib/stormtypes.py

```diff
@@ -4808,6 +4808,22 @@ def __pow__(self, othr):
 
     __rpow__ = __pow__
 
+    def __mod__(self, othr):
+        if isinstance(othr, float):
+            othr = s_common.hugenum(othr)
+            return Number(s_common.hugemod(self.value(), othr)[1])
+        elif isinstance(othr, (int, decimal.Decimal)):
+            return Number(s_common.hugemod(self.value(), othr)[1])
+        elif isinstance(othr, Number):
+            return Number(s_common.hugemod(self.value(), othr.value())[1])
+
+        mesg = f"'**' not supported between instance of {self.__class__.__name__} and {othr.__class__.__name__}"
```

Comment:
need to update operator in the error message

Reply:
Good catch, thanks. Fixed.